cmd/backendapi/apisrv/apisrv.go

```diff
@@ -286,7 +328,7 @@ func (s *backendAPI) DeleteMatch(ctx context.Context, mo *backend.MatchObject) (
 		beLog.WithFields(log.Fields{
 			"error":     err.Error(),
 			"component": "statestorage",
-		}).Error("Statestorage error")
+		}).Error("State storage error")
 
 		stats.Record(fnCtx, BeGrpcErrors.M(1))
 		return &backend.Result{Success: false, Error: err.Error()}, err
```

Comment:
Is there a valid way to cancel without triggering the error stat?

Reply:
I'm not sure that cancelling a deletion is a case we are expecting?  Can you elaborate a bit?

Reply:
Ah, this is delete. Nevermind, totally get it!